manifests/m/Microsoft/OpenJDK/17/17.0.2.8/Microsoft.OpenJDK.17.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenJDK.17
+PackageVersion: 17.0.2.8
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-17.0.2.8.1-windows-x64.msi#winget
+  InstallerSha256: 413C0354CB2F231405C8D2BE48847A4499050C15A7F01C3C1F42C6D84F059EE8
+  ProductCode: '{A34F9424-5651-47B0-825B-9C761BB7430B}'
+- Architecture: arm64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-17.0.2.8.1-windows-aarch64.msi#winget
```

Comment:
```suggestion
  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/9b41bfa3-0cc3-4830-aef4-5b861199d54d/2eeac5626a460f38a8e223beba9e1275/microsoft-jdk-17.0.2.8.1-windows-aarch64.msi
```

Reply:
Leaving as-is, we use aka.ms to measure usage.